scripts/docker_run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# printf "\n[+] (Mac) Running Docker Container"
+# docker container run --name devi -d -p 8501:8501 bennywestsyde/gatorminer && echo "\n\tYou can now view your Streamlit app in your browser.\n\n\tNetwork URL: http://localhost:8501 \n\tExternal URL: http://141.195.4.17:8501"
+docker run -it -d -p 8080:8501 gatorminer
```

Comment:
I think that the way that the docker image is being run is inconsistent across scripts.

In the Windows script:

```
docker container run --name devi -d -p 8501:8501 bennywestsyde/gatorminer
```
This command is first checking if the image exists locally and if not, it's trying to download and build it from docker hub. This is a great approach but it also means that the docker build command is not really necessary because the command will take care of it.

On the other hand:
```
docker run -it -d -p 8080:8501 gatorminer
```
this command will check if the image exists locally but it will not be able to find it through docker hub if it doesn't exist locally. This is because the organization is not specified. It will still try to build it if it's not present.

Personally, I think that you should not create the scripts and instead simply list the correct `docker run` command with the organization and repository name in the README file. If you prefer to keep the scripts, I don't think that they should be split up into the `docker_build` and `docker_run` since `docker run` command generally takes care of building the image if it doesn't exist locally. 

Please feel free to ask questions on this comment if you need clarifications

Reply:
The difference is noted. I apologize for the change in the port number. I was using 8080 while 8501 is occupied with another locally running streamlit instance. It is being changed back to 8501 in the latest commits.

I agree with @noorbuchi that there is probably no need for a script file just for the run command. But I do appreciate the separation of build and run, especially that the build command could be more complex if we are dealing with deployment and version releases. Again, open to suggestions, what do you think @BennyWestsyde?

On another note, is there a specific reason @BennyWestsyde that you chose `--name devi` as the container name in the command?